spec/features/views_show_page_spec.rb

```diff
@@ -3,24 +3,45 @@
 feature "views show page", %(
 As a baller,
 I want to see details about a court
-So I can determine if it's worth my time.
+So I can see upcoming meetups.
 ) do
 
   # Acceptance Criteria
   # [X] User can visit show page from index page by clicking the court name
   # [X] Info displayed includes address, hoop_count, paid?, hours
+  # [X] User should see meetups but no links edit or delete them
 
-  scenario "user views all courts" do
-    court_1 = FactoryGirl.create(:court, hours: "9AM-10:30PM")
+  scenario "user views show page" do
+    court = FactoryGirl.create(:court, hours: "9AM-10:30PM")
+    user_1 = FactoryGirl.create(:user)
+    meetup_1 = FactoryGirl.create(:meetup,
+                                  user: user_1,
+                                  court: court,
+                                  description: "Ballin")
+    meetup_2 = FactoryGirl.create(:meetup,
+                                  user: user_1,
+                                  court: court,
+                                  description: "Shot Callin",
+                                  start_time: Time.now + 100000)
+    meetup_3 = FactoryGirl.create(:meetup,
+                                  user: user_1,
+                                  court: court,
+                                  description: "Not hoggin",
+                                  start_time: Time.now + 10000000)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.